homeassistant/components/swiss_public_transport/coordinator.py

```diff
@@ -49,7 +51,17 @@ def __init__(self, hass: HomeAssistant, opendata: OpendataTransport) -> None:
         )
         self._opendata = opendata
 
-    async def _async_update_data(self) -> DataConnection:
+    def remaining_time(self, departure) -> timedelta | None:
+        """Calculate the remaining time for the departure."""
+        departure_datetime = dt_util.parse_datetime(departure)
+
+        return (
+            departure_datetime - dt_util.as_local(dt_util.utcnow())
+            if departure_datetime
+            else None
+        )
```

Comment:
Please avoid multiline if statements

Reply:
The `if` statement is not multiline, but rather the return statement. Do you mean I should not have multiline return statement and rather do:

```
if ...:
    return ...
else:
    return ...
```